run_idempotent_azure.sh

```diff
@@ -36,16 +36,25 @@ modified_slug_module="${modifiedslug_with_sha}=${modified_module}"
 MVNOPTIONS="-Ddependency-check.skip=true -Dmaven.repo.local=$AZ_BATCH_TASK_WORKING_DIR/dependencies_${modifiedslug_with_sha}=${modified_module} -Dgpg.skip=true -DfailIfNoTests=false -Dskip.installnodenpm -Dskip.npm -Dskip.yarn -Dlicense.skip -Dcheckstyle.skip -Drat.skip -Denforcer.skip -Danimal.sniffer.skip -Dmaven.javadoc.skip -Dfindbugs.skip -Dwarbucks.skip -Dmodernizer.skip -Dimpsort.skip -Dmdep.analyze.skip -Dpgpverify.skip -Dxml.skip -Dcobertura.skip=true -Dfindbugs.skip=true"
 
 # echo "================Cloning the project"
-bash $dir/clone-project.sh "$slug" "$modified_slug_module" "$input_container"
+bash $dir/clone-project.sh "$slug" "${modifiedslug_with_sha}=${modified_module}" "$input_container"
```

Comment:
Why is this line changed? Should be fine to just use $modified_slug_module

Reply:
Actually I should have added module='.' and modified-module but somehow it wasn't pushed

Reply:
I still don't understand why you had to change this line in this file and in many other files.
$modified_slug_module is the same as ${modifiedslug_with_sha}=${modified_module} so we should be able to use $modified_slug_module

In fact, even in MVNOPTIONS we should be able to directly use $modified_slug_module instead of ${modifiedslug_with_sha}=${modified_module}

Reply:
I have mixed the words, fixed right now